packages/cli/package.json

```diff
@@ -8,44 +8,45 @@
   },
   "bugs": "https://github.com/paljs/prisma-tools/issues",
   "dependencies": {
-    "@oclif/command": "^1",
-    "@oclif/config": "^1",
+    "@oclif/core": "^1.19.1",
     "@oclif/plugin-help": "^5",
-    "@paljs/create": "^4.1.0",
-    "@paljs/display": "^4.0.0",
-    "@paljs/generator": "^4.1.0",
-    "@paljs/schema": "^4.1.0",
-    "@prisma/sdk": "^3.13.0",
+    "@oclif/plugin-plugins": "^2.1.5",
+    "@paljs/create": "workspace:*",
```

Comment:
YES, I'm pretty sure you've faced the issue with yarn workspaces where it fetches the online version even when you have them locally, with pnpm as you did you just explicitly reference the local workspace packages and when publishing it modifies those to the actual version it's linked to so that it can fetch the online packages in production